dev-infrastructure/modules/metrics/rules/msftPrometheusAlertingRules.bicep

```diff
@@ -1442,10 +1440,10 @@ resource msiCredentialRefresher 'Microsoft.AlertsManagement/prometheusRuleGroups
         }
         annotations: {
           correlationId: 'ClusterCredentialExpiringSoon/{{ $labels.cluster }}'
-          description: 'Cluster credential for cluster #$.labels.cluster# is expiring in less than 30 days. Please follow TSG to diagnose rotation issues.'
+          description: 'Cluster credential for cluster {{ $labels.cluster }} is expiring in less than 30 days. Please follow TSG to diagnose rotation issues.'
           runbook_url: 'https://eng.ms/docs/cloud-ai-platform/azure-core/azure-cloud-native-and-management-platform/control-plane-bburns/azure-red-hat-openshift/azure-redhat-openshift-team-doc/doc/tsgs/credential-refresher-expiring-cert'
           summary: 'Cluster credential expiring in less than 30 days'
-          title: 'One or More Cluster Credentials are Expiring Soon.'
+          title: 'Cluster credential for cluster {{ $labels.cluster }} is expiring in less than 30 days. Please follow TSG to diagnose rotation issues.'
```

Comment:
As this is the title that ends up in IcM, I would recommend not including the "Please follow..." clause.